test/karma.conf.js

```diff
@@ -18,7 +18,29 @@ module.exports = function (config) {
         },
         browserify: {
             debug: true,
-            transform: [['babelify', { 'presets': ['es2015'] }]]
+            transform: [['babelify', {
+                "presets": [
+                    [
+                        "@babel/preset-env",
```

Comment:
those presets for webrtc, they are not equal to es2015. also in karma config we can see lines about ie11, what is contradictionarary here

Reply:
those presets not just for webrtc, but for our current browser compatibility requirements: https://jira.realeyesit.com/confluence/display/DC/Supported+Platforms

I synced karma conf with this preset, IE removed, bs browsers updated for this requirement.